unified-codes/apps/legacy-api/src/app/server.ts

```diff
@@ -8,7 +8,11 @@ const start = async (): Promise<void> => {
     const port = parseInt(process.env.NX_LEGACY_API_PORT);
     await server.listen(port, '0.0.0.0');
   } catch (err) {
-    server.log.error(err);
+    if (server) {
```

Comment:
actually turned out to be not a problem after fixing the build, but seems sensible to retain.